integreat_cms/cms/templates/dashboard/todo_dashboard_rows/_unreviewed_pages_row.html

```diff
@@ -0,0 +1,36 @@
+{% extends "../_todo_dashboard_row.html" %}
+{% load i18n %}
+{% block todo_dashboard_icon %}
+    view
+{% endblock todo_dashboard_icon %}
+{% block todo_dashboard_title_link %}
+    {% url 'pages' region_slug=request.region.slug language_slug=default_language_slug %}?status=REVIEW
+{% endblock todo_dashboard_title_link %}
+{% block todo_dashboard_title %}
+    {% translate "Pages that are waiting for review" %}
+{% endblock todo_dashboard_title %}
+{# djlint:off #}
+{% block todo_dashboard_number %}
+    {{ unreviewed_pages|length }}{% endblock todo_dashboard_number %}
+{# djlint:on #}
+{% block todo_dashboard_description %}
+    {% if unreviewed_pages %}
+        {% blocktranslate trimmed %}
+            The page <b>{{ single_unreviewed_page }}</b> is waiting for a review. Please review, if the content on the page is correct.
```

Comment:
```suggestion
        {% blocktranslate trimmed with single_unreviewed_page=unreviewed_pages.0 %}
            The page <b>{{ single_unreviewed_page }}</b> is waiting for a review. Please review, if the content on the page is correct.
```

Reply:
Do you have any preference of 
```jinja
        {% blocktranslate trimmed with single_unreviewed_page=unreviewed_pages.0 %}
            The page <b>{{ single_unreviewed_page }}</b> is waiting for a review. Please review, if the content on the page is correct.
```
over
```jinja
        {% blocktranslate trimmed %}
            The page <b>{{ unreviewed_pages.0 }}</b> is waiting for a review. Please review, if the content on the page is correct.
```
or is that just because that variable was used earlier?

Reply:
Exactly, I think in general it's better to avoid the definition of new variable names for trivial stuff, but in the translation files it's preferable to have more speaking variable names to get the context for the translation.